src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ClassDataContract.cs

```diff
@@ -56,23 +46,30 @@ private ClassDataContract(Type type, XmlDictionaryString ns, string[] memberName
         private void InitClassDataContract()
         {
             _helper = (base.Helper as ClassDataContractCriticalHelper)!;
-            this.ContractNamespaces = _helper.ContractNamespaces;
-            this.MemberNames = _helper.MemberNames;
-            this.MemberNamespaces = _helper.MemberNamespaces;
-            _isScriptObject = _helper.IsScriptObject;
+            ContractNamespaces = _helper.ContractNamespaces;
+            MemberNames = _helper.MemberNames;
+            MemberNamespaces = _helper.MemberNamespaces;
+        }
+
+        public override DataContract? BaseContract
+        {
+            [RequiresUnreferencedCode(DataContract.SerializerTrimmerWarning)]
+            get => BaseClassContract;
```

Comment:
This is just a style thing, but is there a way to apply the contract using the more compact expression body style? E.g.:
```c#
public override DataContract? BaseContract => BaseClassContract;
```
I'm guessing that might not be possible with the need to apply the attribute. Would it work to have the attribute on the entire property instead of specifically an explicit getter definition?

Reply:
You guessed it. The attribute can't be applied on the entire property. So it's got to stick with the more verbose syntax.